playbooks/roles/rbenv/tasks/main.yml

```diff
@@ -163,6 +163,14 @@
     - install
     - install:base
 
+- name: update rubygems
+  shell: "gem update --system {{ rbenv_rubygems_version }}"
```

Comment:
Why is this block being moved?

Reply:
We are updating bundler with gems. I think it will be good to move the gems update block before bundler.